src/models/room-state.ts

```diff
@@ -408,7 +426,7 @@ export class RoomState extends TypedEventEmitter<EmittedEvents, EventHandlerMap>
      * Fires {@link RoomStateEvent.Events}
      * Fires {@link RoomStateEvent.Marker}
      */
-    public setStateEvents(stateEvents: MatrixEvent[], markerFoundOptions?: IMarkerFoundOptions): void {
+    public setStateEvents(stateEvents: MatrixEvent[], options?: IMarkerFoundOptions): void {
```

Comment:
Not sure why this is changing anymore

Reply:
Its changed back to not have the option to configure isStartOfTimeline when callling setStateEvents.
It is now what it was initially.